atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/CommitLockTest.java

```diff
@@ -215,4 +166,23 @@ private LockResponse lock(LockDescriptor lockDescriptor) {
         LockRequest lockRequest = LockRequest.of(ImmutableSet.of(lockDescriptor), 5_000);
         return timelockService.lock(lockRequest);
     }
+
+    private TransactionManager createTransactionManager(ConflictHandler conflictHandler) {
+        ImmutableMap<TableReference, ConflictHandler> tablesToWriteWrite = ImmutableMap.of(
+                TEST_TABLE, conflictHandler, TransactionConstants.TRANSACTION_TABLE, ConflictHandler.IGNORE_ALL);
+        TestTransactionManagerImpl transactionManager = new TestTransactionManagerImpl(
+                MetricsManagers.createForTests(),
+                keyValueService,
+                inMemoryTimelockExtension,
+                lockService,
+                transactionService,
+                TestConflictDetectionManagers.createWithStaticConflictDetection(tablesToWriteWrite),
+                sweepStrategyManager,
+                timestampCache,
+                MultiTableSweepQueueWriter.NO_OP,
+                knowledge,
+                MoreExecutors.newDirectExecutorService());
+        transactionManager.overrideConflictHandlerForTable(TEST_TABLE, conflictHandler);
```

Comment:
It's a little strange that we are specifying a strategy for the transaction table in line 172 in the map, but not here (as discussed yesterday night, the transaction table is actually being treated as WRITE_WRITE because of the caching layer). We should probably remove the transaction table from line 172.

After that, I'm not sure we need an explicit map as opposed to inlining an `ImmutableMap.of(TEST_TABLE, conflictHandler)` though that is probably a style preference thing, and either way is probably OK